_templates/db.go.tmpl

```diff
@@ -21,7 +21,7 @@ func Connect() *gorm.DB {
 		log.Fatalf("Got error when connect database, the error is '%v'", err)
 	}
 	db.LogMode(false)
-	if os.Getenv("DB") == "DEBUG" {
+	if gin.IsDebugging(){
```

Comment:
Please insert space after `()`.

I strongly recommend to configure your editor to run `go fmt` before saving.